app/controllers/admin/stats_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Admin::StatsController < Admin::BaseController
+
+  def index
+    @stats_controller = Quizathon::StatsController.new
+    @stats_controller.set_user_with_most_quiz(params[:from]&.to_time, params[:to]&.to_time&.+(1.day))
```

Comment:
All these things could be done in a single method like fetch stats and use those accessors.
Just initialize with all params